pkgs/development/ocaml-modules/bencode/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildDunePackage,
+  fetchurl,
+  ounit,
+  qcheck,
+}:
+
+buildDunePackage rec {
+  pname = "bencode";
+  version = "2.0";
+  minimalOCamlVersion = "4.02.0";
+
+  src = fetchurl {
+    url = "https://github.com/rgrinberg/bencode/archive/${version}.tar.gz";
+    hash = "sha256-Iz6uCBcSbpxKeBvxMp2DRnK3eVTmuYOk0KKY0eL/B1Y=";
+  };
```

Comment:
You must use `fetchzip` (or `fetchFromGitHub`) here (and don’t forget to update the hash).